base/ubi9/entrypoint.sh

```diff
@@ -5,6 +5,16 @@ if [ ! -d "${HOME}" ]; then
   mkdir -p "${HOME}"
 fi
 
+# Configure container builds to use vfs or fuse-overlayfs
+if [ ! -d "${HOME}/.config/containers" ]; then
```

Comment:
Nitpicky, but couldn't the outer if block be removed and just be `mkdir -p "${HOME}/.config/containers"` ?